packages/material-ui/src/CircularProgress/CircularProgress.d.ts

```diff
@@ -40,13 +40,13 @@ export interface CircularProgressProps
 
 export type CircularProgressClassKey =
   | 'root'
-  | 'static'
```

Comment:
Removing it is a breaking change. classes and props should only be added.

Reply:
> Removing it

"Forgetting to add it back" 🙂